use-cases/using-Intel-SGX-and-kata.md

```diff
@@ -0,0 +1,68 @@
+# Kata Containers with SGX
+
+- [Install Host kernel with SGX support](#install-host-kernel-with-sgx-support)
+- [Install Guest kernel with SGX support](#install-guest-kernel-with-sgx-support)
+- [Run Kata Containers with SGX enabled](#run-kata-containers-with-sgx-enabled)
+
+Intel® Software Guard Extensions (SGX) is a set of instructions that increases the security
+of applications code and data, giving them more protections from disclosure or modification.
+At the time of writing this document, SGX patches have not landed on the Linux kernel
+project and `cloud-hypervisor` 0.8.0 doesn't support SGX, so specific versions of `cloud-hypervisor`,
+guest and host kernel must be installed to enable SGX.
+
+## Install Host kernel with SGX support
```

Comment:
We probably need a warning here explaining that:

- you need to understand the implications of building your own kernel.
- you'll "lose" SGX support when a new distro kernel is installed and the system rebooted.